README.md

```diff
@@ -25,12 +25,9 @@ OPTIONAL SECTIONS:
 After you're finished please remove all the comments and instructions!
 -->
 
-<div align="center">
-  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
-  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
-  <br/>
 
-  <h3><b>Microverse README Template</b></h3>
+
+  <h3><b> HELLO-MICROVERSE PROJECT README FILE</b></h3>
 
 </div>
```

Comment:
- [ ] Your feature branch should have the name of a feature or action (e.g. 'created_sidebar' or 'added_comments_section'). e.g _setup-hello microverse_. Kindly checkout to a new branch and use the correct name and mention the old PR in the new PR.